src/python/WMCore/ACDC/DataCollectionService.py

```diff
@@ -350,3 +353,32 @@ def getLumilistWhitelist(self, collectionID, taskName):
 
         lumiList = LumiList(compactList=self.getLumiWhitelist(collectionID, taskName))
         return lumiList
+
+
+def getMergedParents(childLFNs):
+    import threading
+    from WMCore.DAOFactory import DAOFactory
```

Comment:
Once it is imported it doesn't really load multiple times. I think it just check the dictionary where module is already loaded. Anyway, I was just thinking this can be just used as library.
But I moved to the top now.